spec/js/views/todo_view_spec.js

```diff
@@ -40,4 +39,33 @@ define([
       expect(instance.$el.find('.delete').length).toBe(expectedLength);
     });
   });
+
+  describe('Check item', function () {
```

Comment:
You are describing "check item" but you are actually calling "toggleCompleteStatus"